.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,71 @@
+---
+name: Bug report
+about: Report a bug in a recent version of Gluon
+label: bug
+---
+
+<!--
+
+Please carefully fill out the questionaire below to help improve the
+timely triaging of issues. Walk through the questions below and use
+them as an inspiration for what information you can provide.
+
+Make use of codeblocks (three backticks before and after) where
+appropriate (configuration excerpts, log output, etc.). Example:
+
+```
+your code goes here
+```
+
+You can use the "Preview" tab to check how your issue is going to look
+before you actually send it in.
+
+Thank you for taking the time to report a bug with the Gluon project.
+
+-->
+
+### Bugreport
+
+**What is the problem?**
+<!-- 
+- What is not working as expected?
+- How is it misbehaving?
+- When did the problem first start showing up?
+- What were you doing when you first noticed the problem?
+- On which devices (vendor, model and revision) is it misbehaving?
+- Does the issue appear on multiple devices or targets?
+-->
```

Comment:
I'm wondering whether having so many questions in a single comment is a good idea. People without a technical background might write their answers inside the comment...

Reply:
I'd say people without any technical background rarely have accounts on GitHub to open an issue in the first place. Adding documentation on how to fill out the bug report inside of HTML comments is quite normal.

I can add another comment that tells the reporter to watch out and not put their text inside a comment, if that helps at all, seeing as we've amassed quite some text already.

Reply:
@neoraider On second thought, if you're only wondering I'd say that we should just give it a try and makes changes as we go. This is only our first issue template after all.